v4/source/users.yaml

```diff
@@ -1541,3 +1542,31 @@
           $ref: '#/responses/Unauthorized'
         '403':
           $ref: '#/responses/Forbidden'
+    get:
+      tags:
+      - users
+      - terms of service
+      summary: Fetches user's latest terms of service action if accepted.
```

Comment:
I don't really understand this. Does it mean that it fetches the latest "accept" action of the terms of service by a user?

Reply:
We only store a user's latest action if it was for accepting the TOS, and delete it on rejection. So the API only returns the accepted action, if the latest action was for acceptance.

Perhaps the updated text makes sense?